src/components/Attachments/AttachmentView/AttachmentViewVideo/index.tsx

```diff
@@ -1,28 +1,22 @@
-import PropTypes from 'prop-types';
 import React from 'react';
+import type {ImageURISource} from 'react-native';
 import VideoPlayer from '@components/VideoPlayer';
 import useThemeStyles from '@hooks/useThemeStyles';
 import useWindowDimensions from '@hooks/useWindowDimensions';
 
-const propTypes = {
+type AttachmentViewVideoProps = {
     /** Video file source URL */
-    source: PropTypes.string.isRequired,
+    source: string | ImageURISource | ImageURISource[];
```

Comment:
```suggestion
    source: string;
```

The source can only be a string. Internally we use `<Video />` from Expo and the source object is defined here https://docs.expo.dev/versions/latest/sdk/av/#avplaybacksourceobject where `uri` is a string